locales/cy.yml

```diff
@@ -3,3 +3,11 @@ cy:
     example:
       name: 'Cyngor ar Bopeth'
       body: 'Dyma ychydig o destun y corff'
+  components:
+    callout:
+      warning:
+        adviser: Adviser
+        example: Example
+        important: Important
+    oisc_warning:
+      title: This is an OISC level 2 topic
```

Comment:
These are currently provided by the CMS so won't be included in this translation file. Need to remove to avoid confusion.